packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.doc.tsx

```diff
@@ -23,6 +24,9 @@ const ComboBoxVirtualizedExampleCodepen = require('!raw-loader!office-ui-fabric-
 const ComboBoxCustomStyledExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.CustomStyled.Example.tsx') as string;
 const ComboBoxCustomStyledExampleCodepen = require('!raw-loader!office-ui-fabric-react/lib/codepen/components/ComboBox/ComboBox.CustomStyled.Example.Codepen.txt') as string;
 
+const ComboBoxRequiredExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Required.Example.tsx') as string;
+const ComboBoxRequiredExampleCodepen = require('!raw-loader!office-ui-fabric-react/lib/codepen/components/ComboBox/ComboBox.Required.Example.Codepen.txt') as string;
```

Comment:
@ecraig12345 would you have a recommendation on how to solve this?

I am adding a codepen for which I added the above line. This gives me a build error Error: Can't resolve 'office-ui-fabric-react/lib/codepen/components/ComboBox/ComboBox.Required.Example.Codepen.txt'. I pretty much went by the other examples, but do not see any of the other .txt files either and am not sure where to check these in.

Reply:
Those files are autogenerated. There may be a list you have to update somewhere, @kenotron would know.

Reply:
The proper way to load codepen components has changed. Merge with master then look at one of the other lines that loads a codepen file to see the new way.

Reply:
thanks @ecraig12345 and @natalieethell